website/content/en/preview/upgrade/upgrade-ref.md

```diff
@@ -0,0 +1,606 @@
+
+## Global Settings
+
+The v1beta1 specification removes the `karpenter-global-settings` ConfigMap in favor of setting all Karpenter configuration using environment variables. Along, with this change, Karpenter has chosen to remove certain global variables that can be configured with more specificity in the EC2NodeClass . These values are marked as removed below.
+
+
+| **`karpenter-global-settings` ConfigMap Key**     | **Environment Variable**        | **CLI Argument**
+|---------------------------------------------------|---------------------------------|-----------------------------------|
+| batchMaxDuration                                  | BATCH_MAX_DURATION              | --batch-max-duration              |
+| batchIdleDuration                                 | BATCH_IDLE_DURATION             | --batch-idle-duration             |
+| aws.assumeRoleARN                                 | AWS_ASSUME_ROLE_ARN             | --aws-assume-role-arn             |
+| aws.assumeRoleDuration                            | AWS_ASSUME_ROLE_DURATION        | --aws-assume-role-duration        |
+| aws.clusterCABundle                               | AWS_CLUSTER_CA_BUNDLE           | --aws-cluster-ca-bundle           |
+| aws.clusterName                                   | AWS_CLUSTER_NAME                | --aws-cluster-name                |
+| aws.clusterEndpoint                               | AWS_CLUSTER_ENDPOINT            | --aws-cluster-endpoint            |
+| aws.defaultInstanceProfile                        | Dropped                         | Dropped                           |
+| aws.enablePodENI                                  | Dropped                         | Dropped                           |
+| aws.enableENILimitedPodDensity                    | Dropped                         | Dropped                           |
+| aws.isolatedVPC                                   | AWS_ISOLATED_VPC                |--aws-isolated-vpc                 |
+| aws.vmMemoryOverheadPercent                       | AWS_VM_MEMORY_OVERHEAD_PERCENT  |--aws-vm-memory-overhead-percent   |
+| aws.interruptionQueueName                         | AWS_INTERRUPTION_QUEUE_NAME     |--aws-interruption-queue-name      |
+| featureGates.enableDrift                          | FEATURE_GATE="Drift=true"       |--feature-gates Drift=true         |
+
+## Drift Enabled by Default
+
+The drift feature will now be enabled by default starting from v0.33.0. If you don’t specify the Drift featureGate, the feature will be assumed to be enabled. You can disable the drift feature by specifying --feature-gates Drift=false. This feature gate is expected to be dropped when core APIs (NodePool, NodeClaim) are bumped to v1.
+
```

Comment:
I don't see mention of the updating sections that were added at the bottom of the draft migration proposal doc. There's a few other things at the bottom of that doc that we should add in here including:
1. AMI Requirements Tagging was dropped
2. Naming on Instances changed to nodeName
3. Logging Configuration is Deprecated
4. Logging Configuration is No Longer Dynamic
5. etc.